awd/cli.py

```diff
@@ -185,30 +186,27 @@ def deposit(
 
             # Retrieve and validate Crossref metadata
             crossref_response = crossref.get_response_from_doi(metadata_url, doi)
-            if crossref.is_valid_response(doi, crossref_response) is False:
+            if article.valid_crossref_metadata(crossref_response) is False:
                 continue
-            article.crossref_metadata = crossref.get_metadata_extract_from(
-                crossref_response.json()
-            )
+            article.crossref_metadata = crossref_response.json()
 
             # Create and validate DSpace metadata from Crossref metadata
-            article.dspace_metadata = crossref.create_dspace_metadata_from_dict(
-                article.crossref_metadata,
+            article.dspace_metadata = article.create_dspace_metadata(
```

Comment:
I think the same is true here, where an `Article` property like `.dspace_metadata` is only set from the flow in the CLI function.

BUT, maybe this was all slated for part of the move to `.process()`?  If so, please feel to disregard.